CRM/Core/Block.php

```diff
@@ -394,16 +394,20 @@ private static function setTemplateShortcutValues() {
     }
 
     $values = [];
-    foreach ($shortCuts as $key => $short) {
-      $values[$key] = self::setShortCutValues($short);
-    }
 
     // Deprecated hook with typo.  Please don't use this!
     CRM_Utils_Hook::links('create.new.shorcuts',
       NULL,
       CRM_Core_DAO::$_nullObject,
       $values
     );
+    if ($values) {
```

Comment:
should this be !empty - I worry about whether the handling for empty arrays are consistent over php versions

Reply:
I've never had an issue with that. [PHP docs](https://www.php.net/manual/en/language.types.boolean.php) on the subject are pretty clear and have never changed afak.

Reply:
ok